### PR TITLE
new device (matter-window-covering): Zemismart MT01

### PR DESCRIPTION
**Pull request by lelandblue**
WWST Certification request to add the Zemismart MT01 window covering device.

### CODE REVIEW

File: drivers/SmartThings/matter-window-covering/fingerprints.yml [hunk 1]
```diff
@@ -5,7 +5,12 @@ matterManufacturer:
     vendorId: 0x130A
     productId: 0x55
     deviceProfileName: window-covering-profile
-
+#Zemismart
+- id: 5020/65534
```
Comment (ctowns) [inline drivers/SmartThings/matter-window-covering/fingerprints.yml:9]: Indent this line to follow the formatting in the other fingerprints. Also this should be a string (should be surrounded with quotes "").

Where did this ID come from? I am wondering if we can change it to something more human readable? `5020/65534` doesn't really tell us much about what this device is. See above where the id is "Eve MotionBlinds", which makes it clear what we are looking at.

Comment (lelandblue) [reply drivers/SmartThings/matter-window-covering/fingerprints.yml]: The partner provided that information, I will need to follow up with the certification tool team regarding that. Thanks Cooper. I made another commit.

**Review by ctowns — APPROVED** [review]

File: drivers/SmartThings/matter-window-covering/fingerprints.yml [hunk 2]
```diff
@@ -5,7 +5,12 @@ matterManufacturer:
     vendorId: 0x130A
     productId: 0x55
     deviceProfileName: window-covering-profile
-
+#Zemismart
+  - id: "Zemismart MT01 Slide Curtain"
+    deviceLabel: Zemismart MT01 Slide Curtain
+    vendorId: 0x139C
+    productId: 0xFFFE
+    deviceProfileName: window-covering-battery
```
Comment (ctowns) [inline drivers/SmartThings/matter-window-covering/fingerprints.yml:13]: I believe this should just be `window-covering` (no battery) as per discussion in the latest two comments here: https://smartthings.atlassian.net/browse/MTR-605?focusedCommentId=1416175

**Review by ctowns — CHANGES_REQUESTED** [review]

File: drivers/SmartThings/matter-window-covering/fingerprints.yml (hunk 2 above)
Comment (lelandblue) [reply drivers/SmartThings/matter-window-covering/fingerprints.yml]: Updated, thanks @ctowns

**Review by ctowns — APPROVED** [review]

**Pull request merged**